pkg/controller/jobs/jobset/jobset_webhook.go

```diff
@@ -57,6 +67,34 @@ func (w *JobSetWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	log.V(5).Info("Applying defaults", "jobset", klog.KObj(jobSet))
 
 	jobframework.ApplyDefaultForSuspend(jobSet, w.manageJobsWithoutQueueName)
+	if features.Enabled(features.MultiKueue) {
+		if jobSet.Spec.ManagedBy != nil && *jobSet.Spec.ManagedBy == jobsetapi.JobSetControllerName {
```

Comment:
```suggestion
		if jobSet.Spec.ManagedBy != nil && *jobSet.Spec.ManagedBy != jobsetapi.JobSetControllerName {
```
We want to set `multikueue` controller if: `managedBy == nil || managedBy ==  jobsetapi.JobSetControllerName`, so this should be negation of this condition.